### PR TITLE
[TASK] Add contribution guide

### PR DESCRIPTION
**Pull request by DanielSiepmann**

### CODE REVIEW

**Comment by DanielSiepmann** [comment]
@sbuerk you already approved the original PR, but still I mention you as you kickstarted this topic.

**Comment by DanielSiepmann** [comment]
Also contains info from https://git.typo3.org/qa/internal-organization/-/issues/47

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# How to contribute
+
+## Open an issue
+
+Feel free to open an issue, No matter whether you found a bug, request a feature or have questions.
+
+1. Please check existing open and closed issues first and feel free to comment and re-open them.
```
Comment (oliverklee) [inline CONTRIBUTING.md:7]: I propose we link the issue page on GitHub.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+2. Use a meaningful title
+3. Provide enough context, a concrete use case and steps to reproduce bugs
+
+## Contributing code and documentation
+
+1. Create a fork of the repository (You need a GitHub account)
```
Comment (oliverklee) [inline CONTRIBUTING.md:13]: We can link https://docs.github.com/de/get-started/quickstart/fork-a-repo and https://github.com/join here.

**Review by oliverklee — CHANGES_REQUESTED** [review]
One general remark on the style: We should either use full sentences (uppercase first letter, with a period at the end) or bullet-list-style sentences (lowercase first letter, no period at the end), but not the current mix of "uppercase first letter, but no period at the end".

Also, I propose we link the TYPO3 CoC and document that the author of a PR is responsible for rebasing it (if it doesn't get merged quickly).

File: CONTRIBUTING.md (hunk 2 above)
Comment (DanielSiepmann) [reply CONTRIBUTING.md:13]: I suggest that https://docs.github.com/en/get-started/quickstart/contributing-to-projects already contains the first suggestion.

I'll happily add the 2nd suggestion for joining GitHub.

**Comment by DanielSiepmann** [comment]
> Also, I propose we link the TYPO3 CoC

We already have https://github.com/FriendsOfTYPO3/tea/blob/main/CODE_OF_CONDUCT.md, shouldn't that be enough?

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
+# How to contribute
+
+## Open an issue
+
+Feel free to open an issue, No matter whether you found a bug, request a feature or have questions.
```
Comment (oliverklee) [inline CONTRIBUTING.md:5]:
```suggestion
Feel free to open an issue - no matter whether you found a bug, would like to request a feature or have questions.
```

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+3. Apply your changes.
+4. Commit your changes, follow the commit conventions: https://docs.typo3.org/m/typo3/guide-contributionworkflow/main/en-us/Appendix/CommitMessage.html#commitmessage .
+   Except that we do not have `Releases` or `Change-Id` Metadata.
+5. Push your changes.
+6. Open a pull request.
+
+Some pull requests can not be merged quickly.
+We consider the original author responsible for rebasing open merge requests under that circumstances.
+
+You can find the documented workflow for GitHub at: https://docs.github.com/en/get-started/quickstart/contributing-to-projects .
```
Comment (oliverklee) [inline CONTRIBUTING.md:28]:
```suggestion
Please also have a look at the documented [contribution workflow for GitHub](https://docs.github.com/en/get-started/quickstart/contributing-to-projects).
```

**Comment by oliverklee** [comment]
> We already have https://github.com/FriendsOfTYPO3/tea/blob/main/CODE_OF_CONDUCT.md, shouldn't that be enough?

In theory, people should read both documents. In practice, I think linking from the contribution document to either `CODE_OF_CONDUCT.md` or the TYPO3 CoC probably helps people notice it.

**Review by oliverklee — CHANGES_REQUESTED** [review]

**Review by oliverklee — APPROVED** [review]

**Pull request merged**